ui/app/AppLayouts/Wallet/stores/RootStore.qml

```diff
@@ -197,4 +197,13 @@ QtObject {
     function runEditAccountPopup(address) {
         walletSection.runEditAccountPopup(address)
     }
+
+    function getAllNetworksSupportedString() {
+        let result = ""
+        for(var i = 0; i < allNetworks.count; i++) {
+            let shortName = allNetworks.rowData(i, "shortName")
+            result += shortName + ':'
+        }
+        return result
```

Comment:
You could do this in NIM and export it straight from there :)

Accessing `rowData` from here sounds like a hack... Besides in NIM is gonna be faster

Reply:
done @caybro :)